rfcs/meeting-facillitation.md

```diff
@@ -0,0 +1,27 @@
+# Meeting Facillitation
+
+This document aims to formalize our meeting structure with respect to running them.
+
+-----
+
+Each meeting will have a facillitator and a note taker who sign up for the role for each meeting
+ahead of time. These roles are open to all Community Team members. The responsibilities for each are
+listed below.
+
+### Facillitator
+
+- On the Monday before a team meeting an agenda issue will be issued.
```

Comment:
"issue will be issued" sounds so double. How about "an agenda issue should be created"